### ISSUE TITLE
Example code out of date. UpnpServiceImpl constructor with signature UpnpServiceImpl(RegistryListener) does not exist

### ISSUE THREAD

**Issue by kyle-cackett** [jupnp/jupnp#108]
https://github.com/jupnp/jupnp/blob/64e824434de3d7fa89c8c0af38183ead7e2f680d/docs/gettingstarted.xhtml#L87

**Comment by ijabz** [comment]
Yes the cling source allows a RegistryListener to be passed to UpnpServiceImpl. but that option is missing from the jupnp code., is that a regression or was the code added to cling after jupnp forked from it.

**Comment by kcpants** [comment]
It's been awhile since I looked at this code but, if I remember correctly, I think the issue here is that the example code snippet in the `gettingstarted.xhtml` document does not work because it is out of sync with the actual code in the repository. 

Regardless of whether the `UpnpServiceImpl` constructor which accepts a `RegistryListener` was mistakenly removed from this repo or if it was added to the upstream repo after the fork, the tutorial code contained within this repo references it and therefore either it should exist or the example should be changed.

Looking at the available `UpnpServiceImpl` constructors that do exist within the repo (linked below) I could not find one with the signature `UpnpServiceImpl(RegistryListener)`: 

https://github.com/jupnp/jupnp/blob/8632efeff5024c3ea6472c2df52eaebf78cd1e8d/bundles/org.jupnp/src/main/java/org/jupnp/UpnpServiceImpl.java#L75-L80

https://github.com/jupnp/jupnp/blob/8632efeff5024c3ea6472c2df52eaebf78cd1e8d/bundles/org.jupnp.osgi/src/main/java/org/jupnp/osgi/impl/UPnPServiceImpl.java#L39

This makes me think the correct fix is likely to update `gettingstarted.shtml` with a working example.
